### PR TITLE
Structural 2 - Half Circle to Parabolla

### PR DESCRIPTION
**Pull request by brunovdias**

### CODE REVIEW

File: structural/script.js [hunk 1]
```diff
@@ -300,6 +300,295 @@ const formula_circle_I_y_bar = [
...
+        displayName: 'Calculate area',
+        calculate: (half_cir_r1) => (Math.PI * half_cir_r1**2) / 2
+    },
+    {
+        displayName: 'Calculate r',
+        calculate: (half_cir_area) => (2 * half_cir_area / Math.PI)**(1/2)
+    },
+];
+
+const formula_half_circle_I_x_bar = [
```
Comment (Flopes55) [inline structural/script.js:336]: There is something wrong with this one 
if x bar = 1 then radius = 1.7373686553786964 
if radius =  1.7373686553786964 then x bar = 9.869604401089356

Comment (Flopes55) [inline structural/script.js:336]: There is something wrong with this one 
if x bar = 1 then radius = 1.7373686553786964 
if radius =  1.7373686553786964 then x bar = 9.869604401089356

File: structural/script.js [hunk 2]
```diff
@@ -300,6 +300,295 @@ const formula_circle_I_y_bar = [
...
+        displayName: 'Calculate area',
+        calculate: (quarter_cir_r2) => (Math.PI * quarter_cir_r2**2) / 4
+    },
+    {
+        displayName: 'Calculate r',
+        calculate: (quarter_cir_area) => (4 * quarter_cir_area / Math.PI)**(1/2)
+    },
+];
+
+const formula_quarter_circle_I_x_bar = [
```
Comment (Flopes55) [inline structural/script.js:391]: same with this one
if  I x bar = 1 then radius = 2.0660911663590564
if radius = 2.0660911663590564 then I x bar =  9.869604401089358

File: structural/script.js [hunk 3]
```diff
@@ -300,6 +300,295 @@ const formula_circle_I_y_bar = [
...
+        displayName: 'Calculate I_x_bar',
+        calculate: (quarter_cir_r3) => ((9*(Math.PI**2) - 64) / 144*Math.PI) * quarter_cir_r3**4
+    },
+    {
+        displayName: 'Calculate r',
+        calculate: (quarter_cir_I_x_bar) => ((144*Math.PI*quarter_cir_I_x_bar) / (9*(Math.PI**2) - 64))**(1/4)
+    },
+];
+
+const formula_quarter_circle_I_y_bar = [
```
Comment (Flopes55) [inline structural/script.js:402]: same as previous one

**Review by Flopes55 — CHANGES_REQUESTED** [review]

**Review by Flopes55 — APPROVED** [review]
no errors found

**Comment by teixdavide** [comment]
Overall good. Quarter Elipse I y bar is wrong

**Review by teixdavide — CHANGES_REQUESTED** [review]
Overall good. Quarter Elipse I y bar is wrong

**Review by teixdavide — APPROVED** [review]

**Pull request merged**